### ISSUE TITLE
TypeError: stat: path should be string, bytes, os.PathLike or integer, not NoneType

### ISSUE THREAD

**Issue by Strider26** [paolodragone/pymzn#21]
I am running my code on python 3.6 and have installed the pymzn though pip install
When I tried to run below code it threw an error.

    ```
 import pymzn

 solution = pymzn.minizinc('seq_disjunctive.mzn','Disjunctive.dzn')

```
TypeError: stat: path should be string, bytes, os.PathLike or integer, not NoneType

**Comment by paolodragone** [comment]
It look like no error I've encountered so far. Are you using Windows maybe? It is possible that there are still issues when using PyMzn on Windows.
Also, try to reinstall it now, I might have forgot to upload the last version on PyPI.

If it still does not work, maybe you can send me the minizinc model so I can try to replicate the error.

**Comment by Strider26** [comment]
I am using windows system. I have mailed you the minizinc model.

**Comment by pondinlet** [comment]
On Windows, I found a dependency is missing for appdirs.

pip install appdirs

will fix it.

**Comment by paolodragone** [comment]
Yes, I confirm.
The optional dependence for appdirs was intended, but there is a small bug that will be fixed in the next release. Installing appdirs is a workaround for now.

**Comment by paolodragone** [comment]
Fixed in 0.18.1.
